File: solutions/java/darts/1/src/main/java/Darts.java
class Darts {
    int score(double xOfDart, double yOfDart) {
        double distance = Math.sqrt(Math.pow(xOfDart, 2)+Math.pow(yOfDart, 2));
        if (distance <= 1) {
            return 10;
        }
        if (distance <= 5) {
            return 5;
        }
        if (distance <= 10) {
            return 1;
        }
        return 0;
    }
}

File: solutions/lua/darts/1/darts.lua
local Darts = {}

function Darts.score(x, y)
    local distance = (x*x+y*y)^0.5
    if distance <= 1 then return 10 end
    if distance <= 5 then return 5 end
    if distance <= 10 then return 1 end
    return 0
end

return Darts
